packages/android_alarm_manager/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Remove support for the V1 Android embedding.
 * Updated Android lint settings.
 * Removed `-Werror` in Android builds.
+* Update example app Android compileSdkVersion to 31.
```

Comment:
These should all be "Updates", per the new style guide ("Update" is imperative, rather than indicative).

Hopefully once we've been following the new style for a while it'll get easier, since all the recent entries will start to become consistent over time.